.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,32 @@
+name: Bug report
+description: Create a new issue reporting a bug
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    id: reproduction-steps
+    attributes:
+      label: Reproduction steps
+      description: Steps to reproduce the behavior.
+      placeholder: 1. ...
+        2. ...
+        3. ...
+    validations:
+      required: true
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected behavior
+      description: What did you expect to happen?
+    validations:
+      required: true
+  - type: textarea
+    id: additional-info
+    attributes:
+      label: Additional information
+      description: Add any other context about the problem here e.g. screenshots, logs, etc.
```

Comment:
We can consider adding the Chain ID and Safe Address as optional fields. I'd say for now it's ok to have it as free-form text though and we can check the patterns present in future reports.

Reply:
Agreed. I think it's best to keep it as simple as possible for the time being as well.